### PR TITLE
feat(analytics): expose native automatic screenview reporting toggle

### PR DESCRIPTION
**Pull request by se09deluca**
### Description
See [related issue comment](https://github.com/invertase/react-native-firebase/issues/5131#issuecomment-995212171)

Automatic screenview tracking can now be disabled/enabled via firebase.json throught "google_analytics_automatic_screen_reporting_enabled" property.

Automatic screenview reporting (enabled by default) previously could be disabled trought firebase.json configuration file, but only adding the meta tag in AndroidManifest.xml or Info.plist key (the native way).
This commit includes the needed adds in build.gradle and ios_config.sh to make these config exposed to the react native project.

### Release Summary
Includes google_analytics_automatic_screen_reporting_enabled in firebase.json properties

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- This is a breaking change;
  - [ ] Yes
  - [x] No

### CODE REVIEW

**Review by mikehardy — DISMISSED** [review]
This looks perfect first time! Thank you. I've got two other fixes just merged, hopefully CI goes green on this and I'll release it today

File: docs/analytics/usage/index.md [hunk 1]
```diff
@@ -181,3 +181,18 @@ import { firebase } from '@react-native-firebase/analytics';
 // ...
 await firebase.analytics().setAnalyticsCollectionEnabled(true);
 ```
+
+## Disable screenview tracking
+
+Analytics automatically tracks some information about screens in your application, such as the class name of the UIViewController or Activity that is currently in focus.
+Automatic screenview reporting can be turned off/on through `google_analytics_automatic_screen_reporting_enabled` property of `firebase.json` file.
+
+
```
Comment (mikehardy) [inline docs/analytics/usage/index.md:190]:
```suggestion

```

**Comment by mikehardy** [comment]
Nothing at all wrong with the implementation - I just shepherded it through the trivial lint line-spacing / spellchecking crud 😄  --> merged! 🚀

**Comment by se09deluca** [comment]
Thanks a lot! 
It has been my first contribution, I’m not expert on this. Next time I’ll surely give more attention!
🙏🏻☺️

**Comment by Jbug1984** [comment]
I'm not sure (how to do all this )

**Pull request merged**